tests/array/first.js

```diff
@@ -0,0 +1,15 @@
+var chai = require('chai');
+var expect = chai.expect;
+
+require('../../array/first');
+
+describe('first', function() {
+  it('should grab the first item from the array', function() {
+    expect(["t", "e", "s", "t"].first(1)).to.eql(["t"]);
```

Comment:
even though it'll work, add a test for `.first()` with no arguments @rampage1213